hydrabus_framework/utils/color.py

```diff
@@ -0,0 +1,9 @@
+class Colors:
```

Comment:
Should inherite of Enum

Reply:
Should Herite* ?

Reply:
Done [#b07b0da](https://github.com/hydrabus-framework/framework/commit/b07b0da8d9fbc27d6abbabb04eb0f0afef1a76db)